backend/Dockerfile

```diff
@@ -37,6 +37,8 @@ COPY . /src/
 WORKDIR /src/
 
 RUN npm install
+RUN chown -R 1001:123 "/root/.npm"
+RUN npm run build && python manage.py shell --command="import environs; env = environs.Env(); print(env.str('ENV', 'UNDEFINED').upper())" && python manage.py collectstatic
```

Comment:
the env printing here is a hangover from our debugging yesterday, can/should be removed

Reply:
can do